remoteappmanager/handlers/home_handler.py

```diff
@@ -265,8 +265,12 @@ def _start_container(self, orm_user, app, policy, mapping_id):
         volumes = {}
 
         if mount_home:
-            home_path = os.path.expanduser('~'+user_name)
-            volumes[home_path] = {'bind': '/workspace', 'mode': 'rw'}
+            home_path = os.environ.get('HOME')
```

Comment:
According to http://superuser.com/questions/271925/where-is-the-home-environment-variable-set it's exported by the login process. Which means that it's always present. Question is if what you are doing to overwrite it also exports it to subprocesses... let me check.